meilisearch/index.py

```diff
@@ -1909,21 +1985,18 @@ def update_embedders(self, body: Union[MutableMapping[str, Any], None]) -> TaskI
         Raises
         ------
         MeilisearchApiError
-            An error containing details about why Meilisearch can't process your request. Meilisearch error codes are described here: https://www.meilisearch.com/docs/reference/errors/error_codes#meilisearch-errors
+            An error containing details about why Meilisearch can't process your request.
+            Meilisearch error codes are described here: https://www.meilisearch.com/docs/reference/errors/error_codes#meilisearch-errors
         """
-
-        if body:
-            for _, v in body.items():
-                if "documentTemplateMaxBytes" in v and v["documentTemplateMaxBytes"] is None:
```

Comment:
Is this handling done by Meili now?

Reply:
Removing it did not trigger any test failure but it might simply be untested, so I added it back to avoid any unwanted side effects